dagger/utilities/dbt_config_parser.py

```diff
@@ -0,0 +1,131 @@
+import json
+from os import path
+from os.path import join
+from pprint import pprint
+from typing import Tuple, List, Dict
+
+import yaml
+
+ATHENA_TASK_BASE = {"type": "athena"}
+S3_TASK_BASE = {"type": "s3"}
+
+
+
+class DBTConfigParser:
+    """
+    Module that parses the manifest.json file generated by dbt and generates the dagger inputs and outputs for the respective dbt model
+    """
+
+    def __init__(self, default_config_parameters: dict):
+        self._default_data_bucket = default_config_parameters["data_bucket"]
+        self._dbt_project_dir = default_config_parameters.get("project_dir", None)
+        dbt_manifest_path = path.join(self._dbt_project_dir, "target", "manifest.json")
+        self._dbt_profile_dir = default_config_parameters.get("profile_dir", None)
```

Comment:
the `_dbt_profile_dir` is duplicated here 👀

Reply:
one is `profile_dir` and the other is `project_dir` 🙈

Reply:
oh!!!! haha > <